docs/book/src/tasks/experimental-features/cluster-class/change-clusterclass.md

```diff
@@ -1,32 +1,192 @@
 # Changing a ClusterClass
 
-When you change a ClusterClass, the system validates the required changes according to the [compatibility rules defined in the ClusterClass proposal](https://github.com/kubernetes-sigs/cluster-api/blob/main/docs/proposals/202105256-cluster-class-and-managed-topologies.md#clusterclass-compatibility).
+## Selecting a strategy
 
-According to [Cluster API operational practices](https://cluster-api.sigs.k8s.io/tasks/updating-machine-templates.html), the recommended way for updating templates is by template rotation (create a new template, update the template reference in the ClusterClass, and then delete the old template).
+When planning a change to a ClusterClass, users should always take into consideration
+how those changes might impact the existing Clusters already using the ClusterClass, if any.
+
+There are two strategies for defining how a ClusterClass change rolls out to existing Clusters: 
+
+- Roll out ClusterClass changes to existing Cluster in a controlled/incremental fashion.
+- Roll out ClusterClass changes to all the existing Cluster immediately.
+
+The first strategy is the recommended choice for people starting with ClusterClass; it
```

Comment:
Just a general comment (I think we don't have to change anything because of that):

In my opinion it's good to keep sentences relatively short (max 1,5-2 lines) because then they are easier to digest/understand for readers. A sentence just becomes harder to understand when there are multiple "subordinate clauses" (googled that word, I hope it fits :)). (Not trying to mandate a hard upper limit, at some point sentences just feel too long to me, usually when I have to reread the sentence to figure out how the clauses fit together)

Usually it's relatively easy to split a long sentence in two (I'm trying to do this everywhere since somebody told me that issue)

Reply:
that's a good suggestion!
I like the direction where this is going, but sometimes is hard to follow

Reply:
Yup absolutely :)